docs/resources/Emoji.md

```diff
@@ -0,0 +1,60 @@
+# Emoji Resource
+
+>warn
+> Routes for controlling emojis do not follow the normal rate limit conventions. These routes are specifically limited on a per-guild basis to prevent abuse. This means that the quota returned by our APIs may be inaccurate, and you may encounter 429s.
+
+### Emoji Object
+
+###### Emoji Structure
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id | snowflake | [emoji id](#DOCS_REFERENCE/image-formatting) |
+| name | string | emoji name |
+| roles | array of [role object](#DOCS_PERMISSIONS/role-object) ids | roles this emoji is whitelisted to |
+| require\_colons | bool | whether this emoji must be wrapped in colons |
+| managed | bool | whether this emoji is managed |
+
+
+## List Guild Emojis % GET /guilds/{guild.id#DOCS_GUILD/guild-object}/emojis
+
+Returns a list of [emoji](#DOCS_EMOJI/emoji-object) objects for the given guild.
+
+## Get Guild Emoji % GET /guilds/{guild.id#DOCS_GUILD/guild-object}/emojis/{emoji.id#DOCS_EMOJI/emoji-object}
+
+Returns a list of [emoji](#DOCS_EMOJI/emoji-object) object for the given guild and emoji IDs.
```

Comment:
This should be "Returns an [emoji](#DOCS_EMOJI/emoji-object) object" since it doesn't return a list.